### PR TITLE
feat: added alerts component

### PR DESCRIPTION
**Pull request by TejasShekar**
Screenshot 2022-02-08 174315

### CODE REVIEW

**Review by zabihshaik — COMMENTED** [review]
Everything Looks Good to me.

File: Components/Alerts/alerts.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.alert{
```
Comment (manujagobind) [inline Components/Alerts/alerts.css:1]: Use space before `{`

File: Components/Alerts/alerts.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+.alert-success{
+    background-color: var(--success);
+
+}
+
+.alert-danger{
+    background-color: var(--danger);
+
```
Comment (manujagobind) [inline Components/Alerts/alerts.css:25]: Remove extra blank lines

**Review by manujagobind — APPROVED** [review]

File: Components/Alerts/alerts.css (hunk 2 above)
Comment (TejasShekar) [reply Components/Alerts/alerts.css:25]: I think prettier hasn't done its job well 😅. I'll fix it.

File: Components/Alerts/alerts.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Tejas Shekar" />
+    <title>Prime8 | Alerts</title>
+    <link rel="stylesheet" href="../../styles.css" />
```
Comment (MaruthiPothuganti) [inline Components/Alerts/alerts.html:9]: Better to import the font in CSS files rather than linking in every HTML file

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended changes: Link of font styles using import in CSS

File: Components/Alerts/alerts.html (hunk 3 above)
Comment (TejasShekar) [reply Components/Alerts/alerts.html:9]: Will do that.

**Pull request merged**